README.md

```diff
@@ -291,22 +145,26 @@ order:
 - Tomáš Vojnar
 - Petr Šilling
 - Pavol Žáčik
+- František Nečas
+- Tomáš Kučma
 - Lukáš Petr
 - Tatiana Malecová
 - Jakub Rozek
 
 ## Publications and talks
 
 There is a number of publications and talks related to DiffKemp:
+
+- NETYS'22 talk  
```

Comment:
I'd suggest keeping NETYS at the bottom, even though it is the newest. Custom patterns are quite specific and advanced and I think it's better to have a general introduction (the ICST paper and devconf talk) at the top.